backend/app/config/settings.py

```diff
@@ -0,0 +1,11 @@
+# All Constants defined here
+
+from pydantic import BaseSettings
+
+
+class Settings(BaseSettings):
+    # Collections
+    COMPANY_COLLECTION: str = "companies"
```

Comment:
What is this for?

Reply:
as we grow in complexity, we will organize and store all magic variables here that pertain to the backend